lib/internal/errors.js

```diff
@@ -86,3 +86,5 @@ module.exports = exports = {
 // Note: Please try to keep these in alphabetical order
 E('ERR_ASSERTION', (msg) => msg);
 // Add new errors from here...
+E('ERR_UNK_FILE_OPEN_FLAG', 'Unknown file open flag: %s');
```

Comment:
Although `util.format` strings should be more readable, and since we are throwing errors performances probably is not the primary concern here..